src/scss/3_components/navigation/_breadcrumb.scss

```diff
@@ -15,9 +15,28 @@
 .breadcrumb {
   @include adjust-font-size-to(13px);
   margin: 1rem;
-  border-bottom: #eee 1px solid;
+  border-bottom: $breadcrumb-border-bottom 1px solid;
 
   @media screen and (min-width: $lap) {
     margin: 1rem 0;
   }
+
+  .breadcrumb--list {
+    list-style: none;
+    margin: 0;
+  }
+
+  .breadcrumb--item {
```

Comment:
incorporate this rule into the `.breadcrumb` rule using the &-- parent selector method - just keeps the code consist